event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/EvaluateNviCandidateWithSyntheticDataTest.java

```diff
@@ -210,9 +209,39 @@ void shouldThrowParsingExceptionWhenChannelIsMalformed() {
         .contains("Required field 'scientificValue' is null");
   }
 
+  // TODO: Replace with test on revision filtering in NP-50261
+  @Test
+  void shouldIncludeRevisionFieldWithSingleChannel() {
+    var publication =
+        factory
+            .withContributor(verifiedCreatorFrom(nviOrganization))
+            .withPublicationChannel(JOURNAL_TYPE, LEVEL_ONE)
+            .withRevisionStatus("Unrevised")
+            .getExpandedPublication();
+
+    var json = publication.toJsonString();
+    assertThat(json).containsIgnoringWhitespaces("\"revision\": \"Unrevised\"");
+  }
+
+  // TODO: Replace with test on revision filtering in NP-50261
+  @Test
+  void shouldIncludeRevisionFieldForAnthology() {
+    var publication =
+        factory
+            .withContributor(verifiedCreatorFrom(nviOrganization))
+            .withPublicationChannel(PUBLISHER_TYPE, LEVEL_ONE)
+            .withPublicationChannel(SERIES_TYPE, LEVEL_UNASSIGNED)
+            .withRevisionStatus("Revised")
+            .getExpandedPublication();
+
+    var json = publication.toJsonString();
+    assertThat(json).containsIgnoringWhitespaces("\"revision\": \"Revised\"");
+  }
+
```

Comment:
Temporary tests to ensure the new field is actually added to test data. Can be replaced with tests for the actual behavior we want in the next PR.